client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEdit.tsx

```diff
@@ -64,31 +63,51 @@ export const StocktakeLineEdit: FC<StocktakeLineEditProps> = ({
     return true;
   };
 
-  const parseError = (error: string): LocaleKey => {
-    switch (true) {
-      case error.indexOf('AdjustmentReasonNotProvided') !== -1:
-        return 'error.provide-reason';
-      case error.indexOf('StockLineReducedBelowZero') !== -1:
-        return 'error.reduced-below-zero';
-    }
-    return 'error.cant-save';
-  };
-
   const onOk = async () => {
-    try {
-      await save(draftLines);
-      if (item) {
-        const highlight = {
-          animation: 'highlight 1.5s',
-        };
-        const rowIds = draftLines.map(line => line.id);
-        rowIds.forEach(id => setRowStyle(id, highlight));
+    const result = await save(draftLines);
+
+    result.batchStocktake?.insertStocktakeLines?.map(response => {
+      if (
+        response.response?.__typename === 'InsertStocktakeLineError' &&
+        response.response?.error?.__typename === 'StockLineReducedBelowZero'
+      ) {
+        return error(t('error.stocktake-has-stock-reduced-below-zero'))();
```

Comment:
I think for there are two types of messages, one when finalising stocktake and the other when editing stocktake item (was seeing finalised messaging when editing stocktake item)

Reply:
I'm sorry... I don't understand what you mean...

Reply:
When I edit a stock line, and there is an issue with reduction below zero, I would get `stocktake-has-stock-reduced-below-zero` message which is `Stock take cannot be finalised because there are stock lines which exist in new outbound shipments`, i think that should be  `reduced-below-zero` ¯\_(ツ)_/¯

Reply:
[763e9a6](https://github.com/openmsupply/open-msupply/pull/1200/commits/763e9a609f8b349a820466ced48b0fbb05123395)